resources/js/pages/library/home.vue

```diff
@@ -9,7 +9,7 @@
           <th>Status</th>
           <th>Action</th>
         </tr>
-        <tr v-for="(result, index) in borrowed" :key="`br-${index}`">
+        <tr v-for="(result, index) in borrowed" :key="index">
```

Comment:
@coderabbit would you expect this to be a good practice in Vue Js? I am not sure if setting the Key the array index is a good idea.

Reply:
@coderabbitai would you expect this to be a good practice in Vue Js? I am not sure if setting the Key to the array index is a good idea.